assets/template/tests/lib.rs

```diff
@@ -32,7 +32,11 @@ fn test_hello() {
     // Test the `free_token` method.
     let manifest = ManifestBuilder::new()
         .call_method(component, "free_token", manifest_args!())
-        .deposit_batch(account)
+        .call_method(
```

Comment:
Ah, I was just copying and pasting the `hello-world` example.

I saw the diff, but didn't bother to change, as it would be an example to demo `ENTIRE_WORKTOP`.

BTW we should probably rename `.deposit_batch` to`.deposit_batch_with_entire_worktop`.

Reply:
Yes, I agree on the naming... It's on my list of tweaks I want to make in scrypto when I get a chance, but feel free to make it yourself :)